projects/palindrome/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Palindrome in Every Language
+layout: default
+date: 2020-10-07
+last-modified: 2020-10-07
+featured-image: 
+tags: [panlindrome]
+authors:
+  - anohene1
+---
+
+This is an article on how to check if a number is a palindrome in every language.
+
+## Description
+
+A palindrome is a number that reads the same backward and forward.
+Example: 343, 121, 909, 222
+
+## Requirements
+
+Create a file called "Even Odd" using the naming convention appropriate for your language of choice.
+Write a program that accepts an input of an integer and prints whether the number is a palindrome or not.
+
+## Testing
+The following table contains various test cases that you can use to verify the correctness of your solution:
+
+| Description                  | Input | Output |
+|------------------------------|-------|--------|
+| no input                     | None  | Usage: please input a number |
+| empty input                  | ""    | Usage: please input a number |
+| invalid input: not a number  | a     | Usage: please input a number |
+| sample input: palindrome     | 232   | Number is a palindrome       |
```

Comment:
I'm wondering if we could simplify this a bit more by outputting true/false directly.
```suggestion
| sample input: palindrome     | 232   | true       |
```

Reply:
The actual code returns "Number is a palindrome" or "Number is not a palindrome", but I can change it to true or false if you wish.

Reply:
Yeah, this would make this program consistent with the search algorithms: https://sample-programs.therenegadecoder.com/projects/binary-search/

In general, it'll relieve a lot of headaches related to testing (e.g. someone spells the string wrong)

Reply:
I have changed them to true or false.